src/ast.mli

```diff
@@ -1,7 +1,83 @@
-type operator = Add | Sub | Mul | Div
+type bin_op =
+    | Add    | Minus | Multiply | Divide
```

Comment:
I don't think you need the first `|`. Same for many declarations below:
- `type types`
- `type bit_op`
- `type actor_op`
- `type expr`
- `type stmt`